runtime/shared_common/CacheMap.cpp

```diff
@@ -567,6 +567,7 @@ SH_CacheMap::startup(J9VMThread* currentThread, J9SharedClassPreinitConfig* pico
 						}
 						if (0 == _sharedClassConfig->readOnlyCacheRuntimeFlags) {
 							_sharedClassConfig->readOnlyCacheRuntimeFlags = (_sharedClassConfig->runtimeFlags | J9SHR_RUNTIMEFLAG_ENABLE_READONLY);
+							_sharedClassConfig->readOnlyCacheRuntimeFlags &= ~J9SHR_RUNTIMEFLAG_AUTOKILL_DIFF_BUILDID;
```

Comment:
Outside of layered caches, does this introduce any change in behavior?

Reply:
No